### DISCUSSION TITLE
Handle different keyboards

### DISCUSSION THREAD

**Discussion by Badsheep** [Hammerspoon/hammerspoon#3420] [Q&A]
Hello,

I'd like to try to replace Karabiner with Hammerspoon, but I'd like to know if is it possible to assign different keyboard shortcuts according different keyboards ? Here is an exemple of what I did in Karabiner : 

"Command-T" on my Logitech Harmony Remote (which is seen bye mac OS as a Bluetooth keyboard) becomes "Ctrl-Shift-T" and is stroked to the system.
But on may main keyboard (Apple) the shortcut is still "Crtrl-Shift-T".

Thanks !

**Comment by dasmurphy** [comment]
That would be lovely to get more informations if this is possible. It should be at least possible to check with `hs.eventtap.new` if something if coming from an external keyboard as far as i know. I can validate after the weekend.

**Comment by Badsheep** [comment]
Thanks ! What more informations do you need ?
My goal is to set different actions depending on which keyboard the shorcut os stroked.

**Reply by dasmurphy** [reply]
I don't need more informations. My problem is that i don't have a Mac at work.

**Reply by Badsheep** [reply]
OK, I'll try to do some tests here with `hs.eventtap.new` :) thanks

**Comment by Badsheep** [comment]
I found [this](https://github.com/Hammerspoon/hammerspoon/issues/1083) in the issues...
I don't think it' possible unfortunately.
